packages/story-editor/src/components/library/panes/text/karma/textSets.cuj.karma.js

```diff
@@ -88,10 +88,7 @@ describe('CUJ: Text Sets (Text and Shape Combinations): Using Text Sets', () =>
     expect((await getSelection()).length).toBeGreaterThan(1);
   });
 
-  // Disable reason: flakey tests.
-  // See https://github.com/google/web-stories-wp/pull/6162
-  // eslint-disable-next-line jasmine/no-disabled-tests
-  xit('should allow user to drag and drop text set onto page', async () => {
+  it('should allow user to drag and drop text set onto page', async () => {
```

Comment:
This test seems to work now (will rerun it a few times though) -- I think the failure came from always enabling the smart colors at that time and thus taking more time. If it's still flaky, will debug it more though.